addons/mail/static/src/models/composer/composer.js

```diff
@@ -563,6 +581,16 @@ function factory(dependencies) {
                 'mainSuggestedPartners',
             ],
         }),
+        /**
+         * Determine if at least one attachment of those linked to composer is pending.
+         */
+        hasUploadingAttachment: attr({
```

Comment:
This field determines whether some attachments linked to this composer are being uploaded.

_(single sentence => "what", the "why" is not necessary because the what is sufficient and clear enough)_